Library/Homebrew/utils/github.rb

```diff
@@ -688,10 +690,14 @@ def multiple_short_commits_exist?(user, repo, commit)
 
     output, _, status = curl_output(
       "--silent", "--head", "--location",
-      "-H", "Accept: application/vnd.github.sha",
+      "--header", "Accept: application/vnd.github.sha",
       "https://api.github.com/repos/#{user}/#{repo}/commits/#{commit}"
     )
 
-    !(status.success? && output && output[/^Status: (200)/, 1] == "200")
+    return true if status.success?
+    return true unless output
+    return true if output[/^Status: (200)/, 1] != "200"
+
+    false
```

Comment:
```suggestion
    return true unless status.success?
    return true if output.blank?
    
    output[/^Status: (200)/, 1] != "200"
```

I think? Please disagree with me if you don't think I'm right as I'm not 100% sure.

Reply:
I agree with the refactor code; I was wrong and will fix it.